test/fail_compilation/fail4082.d

```diff
@@ -17,18 +17,3 @@ nothrow void test1()
 NEXT:
     ;
 }
-
-/*
-TEST_OUTPUT:
----
-fail_compilation/fail4082.d(32): Error: destructor `fail4082.Bar.~this` is not `nothrow`
-fail_compilation/fail4082.d(32): Error: `nothrow` function `fail4082.test2` may throw
```

Comment:
Hmm, I don't think it is particularly good if there's a divergence on this error between posix and non-posix.  Consistency should really be upholded, otherwise it'll come back to bite.  Particularly as the error message itself is a bit cryptic.

Reply:
Agreed. I'm not sure what to do about it.